<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>282</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Имя:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="telephoneLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Телефон:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="telephoneLineEdit">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Адрес:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="addressLineEdit">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator {
 width: 24px;
 height: 24px;
 }
QCheckBox::indicator:unchecked { image: url(:/images/down.png); }
QCheckBox::indicator:checked { image: url(:/images/up.png); }</string>
         </property>
         <property name="text">
          <string>Дополнительная информация</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nickLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Псевдоним:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nickLineEdit">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mailLabel">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Эл. почта:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mailLineEdit">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
